tests/test_validate_metadata.py

```diff
@@ -181,12 +186,16 @@ def test_invalid_ontology_content(self):
         #     with species ontologyID of "NCBITaxon_9606"
         reference_file = open("../tests/data/ontology_invalid.json", "r")
         reference_issues = json.load(reference_file)
-        reference_file.close()
+
+        serialize_issues(metadata)
+        current_file = open("issues.json", "r").read()
```

Comment:
Given `ast` is needed to parse `"issues.json"`, it seems that file isn't actually JSON.  So renaming the file to something that doesn't end in ".json" would help avoid confusion.  I've previously used `.txt` for files with data parseable via the Abstract Syntax Tree module.

```suggestion
        current_file = open("issues.txt", "r").read()
```

And similarly `git mv issues.json issues.txt`.

(This reminds me that I should do the reverse renaming for data under `/tests/mock_data` -- they were previously Gene models (Python objects), but now are JSON cast from Firestore documents.)

Reply:
No longer applicable due to 78ee848